source/standards/managing-sensitive-information.html.md.erb

```diff
@@ -0,0 +1,394 @@
+
+From that basis, it is expected to be clear from this documentation if any context has changed, if the initial concern has now lapsed or if the threat landscape has been updated.
+
+Where there is remaining ambiguity or a resource owner is not sure if a sensitivity has lapsed, or should be renewed, advice should be sought from the Information Assurance and Security team.
+
+In addition a responsible role must undertake to:
+
+- Ensure they are keeping up to date with security threat assessments. Ensuring their judgement is up to date with the latest program guidance.
+- In the case of leavers or a change in role, they must ensure ownership responsibilities are effectively handed over to a new role.
+
+#### Only committing sensitive information detailed in HANDLING_INSTRUCTIONS.md
+
+The assumption of future disclosure should be a consideration for any commit to a private repo.
+The only sensitive materials committed to private repos should be those described in the HANDLING_INSTRUCTIONS.md.
+Any new sources of sensitivity should be added to the HANDLING_INSTRUCTIONS.md along with the commit.
+
+For any materials not detailed in HANDLING_INSTRUCTIONS.md expectations of what we commit should remain aligned between public and private repos.
+For instance, we would not expect secrets or access tokens to be committed to a private repo, without good reasoning described in HANDLING_INSTRUCTIONS.
+
+This will ensure the HANDLING_INSTRUCTIONS, and their version history, will remain a guide to the repository's current and historic sensitivities.
+Allowing for a speedier audit upon a decision to return to the public domain.
+
+#### Regular reviews
+
+A review period should not be longer than one year. One task of a review should be to examine changes during the period and extend the HANDLING_INSTRUCTIONS.md
+with an audit of if it remains accurate, and any new sensitivities introduced during that period.
+
+If a review acts as a rolling audit it can ensure HANDLING_INSTRUCTIONS.md remains accurate, but also provide confidence checkpoints to reduce the cost of an audit when a repository is being considered for release into the public domain.
+
+#### Split the repository along sensitive lines
+
+Where sensitivity has lapsed and a repository can now be made public, but uncertainty remains about the version history and accuracy of the HANDLING_INSTRUCTIONS.md.
+A decision can be made to split a suitable repository. This could be in the format of:
+
+- Functional changes, abstracting a component off into a new public repository and distributing it from there.
+- Partial transfer, for instance with documentation repositories, transferring files where this is confidence they are not OFFICIAL-SENSITIVE to a new repo and leaving ones where uncertainty remains.
+- Full transfer without version history, full transfer of the state of `main` to a new repository, whilst preserving the old archive as private for `git blame` purposes.
```

Comment:
```suggestion
- Full transfer without version history, full transfer of the state of `main` to a new repository, whilst preserving the old archive as private for `git blame` purposes.

Choice between these options will depend on the nature of the sensitivity we initially guarded against.
It will also depend on if there are historic sensitivites. What is important is to ensure no historic sensitivites can be found in either the content of the commit history, or metadata like commit messages.
```